docs/process/integrating-with-fxa.md

```diff
@@ -4,28 +4,123 @@ title: Integration with FxA
+1. A basic understanding of [OAuth 2.0][oauth] is required.
+2. Register for OAuth credentials at https://oauth-stable.dev.lcip.org/console/login. See [OAuth credentials](#oauth-credentials).
+3. Development servers point to: https://stable.dev.lcip.org
+4. User authentication follows the [OAuth 2.0][oauth] protocol.
+5. If the multiple Service Providers (SPs) are accessed, distinct tokens are created with appropriate scopes for each SP.
+6. [Query parameters](#authorization-query-parameters) are set and validate when redirecting to Firefox Accounts.
+7. [Self hosted email-first](#self-hosted-email-first-flow) flows initialize and propagate top of funnel metrics.
+8. [User data and account notifications are properly](#user-data-hygiene-gdpr) handled and GDPR compliant.
+9. An icon suitable to display in FxA’s [Devices & apps][devices-and-apps] list has been sent to FxA devs.
 
-## For Engineers
+### Preparing for Production
 
-Turnip greens yarrow ricebean rutabaga endive cauliflower sea lettuce kohlrabi amaranth water spinach avocado daikon napa cabbage asparagus winter purslane kale. Celery potato scallion desert raisin horseradish spinach carrot soko. Lotus root water spinach fennel kombu maize bamboo shoot green bean swiss chard seakale pumpkin onion chickpea gram corn pea. Brussels sprout coriander water chestnut gourd swiss chard wakame kohlrabi beetroot carrot watercress. Corn amaranth salsify bunya nuts nori azuki bean chick
+1. Register for a production OAuth credentials by filing a [deployment bug][deployment-bug]. See [OAuth credentials](#oauth-credentials).
+2. Production servers point to with https://oauth.accounts.firefox.com.
+3. Someone from the FxA team has reviewed the integration code and tested the flow.
 
-## For Designers
+### User Authentication with OAuth 2.0 / OpenID Connect in a nutshell
 
-Turnip greens yarrow ricebean rutabaga endive cauliflower sea lettuce kohlrabi amaranth water spinach avocado daikon napa cabbage asparagus winter purslane kale. Celery potato scallion desert raisin horseradish spinach carrot soko. Lotus root water spinach fennel kombu maize bamboo shoot green bean swiss chard seakale pumpkin onion chickpea gram corn pea. Brussels sprout coriander water chestnut gourd swiss chard wakame kohlrabi beetroot carrot watercress. Corn amaranth salsify bunya nuts nori azuki bean chick
+1. Create a state token, associate it with a local session.
+2. Send [/authentication request](#authorization-query-parameters) to Firefox Accounts. Upon completion, Firefox Accounts redirects back to your app with state and code.
+3. Confirm returned state token by comparing it with state token associated with the local session.
+4. Exchange code for an access token and possibly a refresh token (for clients that request offline use).
+5. Fetch user profile information with access token from the FxA Profile Server.
+6. Associate profile information with local session and possibly create an account.
```

Comment:
"create an account" seems ambiguous, maybe "create a record for the account in application database" or similar?